src/jdk.httpserver/share/classes/com/sun/net/httpserver/Authenticator.java

```diff
@@ -74,7 +72,7 @@ public Failure (int responseCode) {
         /**
          * returns the response code to send to the client
```

Comment:
I'd prefer to make this a full sentence: 
`* Returns the response code to send to the client.`

Reply:
Change made in https://github.com/openjdk/jdk/pull/301/commits/604ea45ca36c8fb0cb195c581f822dc5ae1c23c1